proto/options.proto

```diff
@@ -155,6 +155,8 @@ message Options {
     current = 0;
     depart_at = 1;
     arrive_by = 2;
+    // Note: temporary until bidirectional a* does time dependence, does not vary the time as the route progresses
+    invariant = 3;
```

Comment:
Is this truly temporary or is there a possibility this would be a useful request option even after bidir a* gets time dependence? This could be useful to a user who wants to say "don't try to predict the future, just assume current conditions will stay the same throughout my route." Just a thought.

Reply:
good question! i dont know. you're right, it doesnt have to be temporary it could be useful indefinitely

Reply:
lets keep it!